### PR TITLE
adjust title, description order

### PR DESCRIPTION
**Pull request by kbhawkey**
Correct order of title and description in pages.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/concepts/containers/
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/concepts/overview/
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/concepts/cluster-administration/
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/concepts/extend-kubernetes/

Note: Not sure if these changes are compatible with the task pages.

**Comment by kbhawkey** [comment]
Yes, a tweak still needed for the task pages.

**Comment by tengqm** [comment]
@kbhawkey  This change looks good to me.
When you said "the issue", were you referring to a specific issue to close with this PR?
/lgtm

**Comment by tengqm** [comment]
alright, #22114 is the issue.

**Comment by kbhawkey** [comment]
I think I just resolved it with last commit. PTAL.

**Comment by kbhawkey** [comment]
Single pages look better:
https://deploy-preview-22113--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/

File: layouts/partials/docs/content-page.html [hunk 1]
```diff
@@ -5,8 +5,7 @@
     Edit This Page
   </a>
 </p>
-{{ if not .page.Params.notitle }}
```
Comment (tengqm) [inline layouts/partials/docs/content-page.html:8]: hey ... the `notitle` param is still relevant in some cases, right?

Comment (kbhawkey) [reply layouts/partials/docs/content-page.html:8]: 👀

Comment (kbhawkey) [reply layouts/partials/docs/content-page.html:8]: I had searched for `notitle` in the site, but perhaps I missed something.
Just added this check back.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**